project/MimaExcludes.scala

```diff
@@ -771,7 +771,14 @@ object MimaExcludes {
         ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.annotation.AlphaComponent"),
         ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.annotation.Experimental"),
         ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.annotation.DeveloperApi")
+      ) ++ Seq(
+        // [SPARK-14483][WEBUI] Display user name foreach job and query
+        ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.scheduler.SparkListenerJobStart.copy"),
```

Comment:
I think we can add the user field to JobStart without breaking backward compatibility. Please refactor the change such that mima excludes are not needed as much as possible.